.teamcity/_self/projects/WebApp.kt

```diff
@@ -584,6 +585,48 @@ object CheckCodeStyleBranch : BuildType({
 	}
 })
 
+object Translate : BuildType({
+	name = "Translate"
+	description = "Extract translatable strings from the source code and build POT file"
+
+	artifactRules = """
+		translate => translate
+	""".trimIndent()
+
+	steps {
+		bashNodeScript {
+			name = "Prepare environment"
+			scriptContent = """
+				# Install modules
```

Comment:
I would recommend just removing all the steps which do things, and just have one with like "hello world"

That way, you can merge this quickly so TeamCity picks up on the configuration, and then in the next PR, we can actually test that these steps work in the branch

Reply:
What Noah has said is a good approach - unless you are fairly sure that the steps work, in which case then let's go ahead and merge it! 😛 

Even in the current form, the worst it will do is the new `Translate` job will simply fail to run, I think?

Reply:
True! I think the only downside to that is people might ping the channel wondering why there's a failing build on their branch, which isn't the end of the world :)

Reply:
Thanks, I followed your advice and replaced the build steps with a placeholder. It's better to not risk causing noise from "failing" builds if there's no benefit and can be avoided :)